docs/content/en/docs/migrate/strategy/_index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Migration strategy
+description: General guidelines for migrating your deployment to KLT
+weight: 10
+hidechildren: false # this flag hides all sub-pages in the sidebar-multicard.html
+---
+
+> **Note**
+This section is under development.
+Information that is published here has been reviewed for technical accuracy
+but the format and content is still evolving.
+We hope you will contribute your experiences
+and questions that you have.
+
+No two migrations are alike but these are some general guidelines
+for how to approach the project:
+
+1. Create a new Kubernetes cluster that is not running anything else
+   and use that to build out your deployment environment.
+1. Install and configure the deployment tool(s) you want to use
+   to deploy the components of your software.
+   You can use different deployment tools for different components.
+1. Install KLT.
+
+TODO: For migration, is it best to try to do all the applications at once
+or should they do one application then move onto the next?
+
+TODO: Similarly, would they start out doing one pre-deployment evaluation,
```

Comment:
For the pre and post, I would suggest doing neither first. Install the KLT operator and run some regular deployments. Watch as KLT creates traces. Become comfortable with what and how it works.

Then add a POST deployment task. If a post deploy task fails, the worst-case is that whatever you're doing in the task just won't happen.

If you do a pre-deploy task and it fails, you run the risk of blocking your deployment.

When you're comfortable with post-deploy tasks, move to pre-deploy tasks.

Reply:
Added as comment to source file.